src/styles/components/markdown-editor-preview.styl

```diff
@@ -0,0 +1,2 @@
+.markdown-editor-preview
+  display none
```

Comment:
Having a preview would be nice I think. But I'm not sure it works properly. The property `previewing` is false by the default, so I don't understand why it was showing anyway, without this css.